app/instance-initializers/ember-intl.js

```diff
@@ -10,6 +10,7 @@ import { filterBy } from 'ember-intl/utils/initialize';
 export function instanceInitializer(instance) {
     let service = instance.container.lookup('service:intl');
 
+    service.set('appInstance', instance);
```

Comment:
That's what it was before the change. I thought I was getting errors from it but maybe it was related to something else instead.